webpack.config.js

```diff
@@ -45,7 +45,7 @@ const config = {
 							loader: 'sass-loader',
 							query: {
 								includePaths: [ 'editor/assets/stylesheets' ],
-								data: '@import "variables";',
+								data: '@import "variables"; @import "mixins";',
```

Comment:
Apologies for being daft here, but I can't find what you're referring to. It should be `_variables.scss` and `_mixins.scss` and `main.scss`, correct? 

<img width="182" alt="screen shot 2017-04-18 at 09 41 31" src="https://cloud.githubusercontent.com/assets/1204802/25119628/3aa36b02-241b-11e7-8b95-e7f10c939db0.png">